c2corg_ui/static/js/slideinfo.js

```diff
@@ -4,17 +4,26 @@ goog.require('app');
 
 
 /**
+ * @param {angular.$compile} $compile Angular compile service.
+ * @param {angular.$templateCache} $templateCache service
  * @return {angular.Directive} Directive Definition Object.
+ * @ngInject
  */
-app.slideInfoDirective = function() {
+app.slideInfoDirective = function($compile, $templateCache) {
+
+  var template = function() {
+    return app.utils.getTemplate('/static/partials/slideinfo.html', $templateCache);
```

Comment:
The recommended way is to use the usual ``template`` or ``templateUrl`` directive parameters whenever possible (standard angular way).
For cards and suggestions (simple search) it was not possible because the partial to use depends on a variable.
In the current case, it seems you don't need to do so. So it's probably better to use ``templateUrl``.